AFNetworking/AFHTTPClientSKZ.m

```diff
@@ -674,6 +674,19 @@ - (void)enqueueBatchOfHTTPRequestOperations:(NSArray *)operations
     [self.operationQueue addOperation:batchedOperation];
 }
 
++ (dispatch_queue_t)backgroundQueue
+{
+    static dispatch_once_t pred;
+    static dispatch_queue_t backgroundQ;
+    
+    dispatch_once(&pred, ^{
+        backgroundQ = dispatch_queue_create("background_http_queue", DISPATCH_QUEUE_SERIAL);
```

Comment:
Since we don't supply a queue, all the current AFNetworking callbacks occur on the main queue (which is serial). So the immediate effect of this change is to offload some work off the main thread and on to a different serial queue (which should improve responsiveness). 

The practical need is that we cannot make a synchronous save context call on the main queue, so switching to a different queue will permit this.